docs/current/cookbook/snippets/cache-dependencies/main.py

```diff
@@ -0,0 +1,43 @@
+import sys
+
+import anyio
+
+import dagger
+
+
+async def main():
+    config = dagger.Config(log_output=sys.stdout)
+
+    async with dagger.Connection(config) as client:
+        # use a python:3.11 container
+        # mount the source code directory on the host
+        # at /src in the container
+        # mount the cache volumes to persist dependencies
+        source = (
+            client.container()
+            .from_("python:3.11")
+            .with_directory(
+                "/src",
+                client.host().directory("."),
+                exclude=[".venv/", ".cache/", "ci/"],
```

Comment:
These excludes aren't in the others (e.g., `node_modules` in Node.js). I'd remove them to not make assumptions on how the code is structured.

Reply:
Modified